src/main/kotlin/no/nav/k9/jobber/Statistikkjobb.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.k9.jobber
+
+import no.nav.k9.domene.repository.StatistikkRepository
+import org.slf4j.LoggerFactory
+import java.time.temporal.ChronoUnit
+import java.util.concurrent.TimeUnit
+import kotlin.concurrent.fixedRateTimer
+
+class Statistikkjobb(private val statistikkRepository: StatistikkRepository) {
```

Comment:
Må gjerne lage en test på dette som inkluderer databasen. Vet ikke hvor lett det er da, har ikke laget det selv for jobber, men det finnes noen andre eksempler som inkluderer database. Kan kanskje også da sjekke ytelsen på det du allerede har.